bflib/src/db/actions.rs

```diff
@@ -1035,6 +1041,64 @@ impl Db {
         )?))
     }
 
+    fn move_cruise_missile<'lua>(
+        &mut self,
+        spctx: &SpawnCtx<'lua>,
+        side: Side,
+        ucid: Option<Ucid>,
+        args: WithPosAndGroup<()>,
+    ) -> Result<Option<GroupId>> {
+        let gid = args.group;
+        let group = group!(self, gid)?;
+        let pos = group_position(spctx.lua(), &group.name)?;
+        let mission = self
+            .cruise_missile_mission(side, ucid, pos, args)
+            .context("generating CruiseMissile mission")?;
+        self.set_ai_mission(spctx, gid, mission)
+            .context("setting ai mission")?;
+        Ok(Some(gid))
+    }
+
+    fn cruise_missile(
+        &mut self,
+        perf: &mut PerfInner,
+        spctx: &SpawnCtx,
+        idx: &MizIndex,
+        side: Side,
+        ucid: Option<Ucid>,
+        name: String,
+        action: Action,
+        args: WithPos<AiPlaneCfg>,
+    ) -> Result<Option<GroupId>> {
+        let gid = self.add_and_spawn_ai_air(
+            perf,
+            spctx,
+            idx,
+            side,
+            &ucid,
+            name,
+            action,
+            0.,
+            &args,
+            None,
+            BitFlags::empty(),
+            move |db, gid, pos| {
+                db.cruise_missile_mission(
+                    side,
+                    ucid,
+                    pos,
+                    WithPosAndGroup {
+                        cfg: (),
+                        pos: args.pos,
+                        group: gid,
+                    },
+                )
+            },
+        )?;
+        self.persisted.deployed.insert_cow(gid);
```

Comment:
Instead of doing this, make the jtac search action groups for artillery. Otherwise, it gets awkward when this group dies, since it's an action group delete_group won't (and shouldn't) remove it from deployed. Also, it could cause big problems elsewhere, because if you find a gid in deployed you expect it's type to be Deployed not Action.